projects/demo/src/modules/experimental/badge-alert/examples/1/index.html

```diff
@@ -0,0 +1,23 @@
+<h4>Desktop</h4>
+<p>
+    <tui-badge-alert>9</tui-badge-alert>
+    <tui-badge-alert size="m">9</tui-badge-alert>
+    <tui-badge-alert size="s">9</tui-badge-alert>
+    <tui-badge-alert size="xs"></tui-badge-alert>
+</p>
+
+<h4>Android</h4>
+<p tuiPlatform="android">
+    <tui-badge-alert>9</tui-badge-alert>
+    <tui-badge-alert size="m">9</tui-badge-alert>
+    <tui-badge-alert size="s">9</tui-badge-alert>
+    <tui-badge-alert size="xs"></tui-badge-alert>
+</p>
+
+<h4>IOS</h4>
+<p tuiPlatform="ios">
+    <tui-badge-alert>9</tui-badge-alert>
+    <tui-badge-alert size="m">9</tui-badge-alert>
+    <tui-badge-alert size="s">9</tui-badge-alert>
+    <tui-badge-alert size="xs"></tui-badge-alert>
```

Comment:
Can we still add 9 in it and hide it with font size? Could be helpful for screen readers.